index.rst

```diff
@@ -48,14 +48,480 @@
+
+.. table:: Visits with inconsistent photometry
+   :name: photometry
+
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | Year | Band | Field | Visits                                         | Plot link                         |
+   +======+======+=======+================================================+===================================+
+   | 2013 | g    | B1    | 210508, 210555, 210597, 210633, 210669         | :numref:`source_counts_2013_B1_g` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2015 | g    | B1    | 427628                                         | :numref:`source_counts_2015_B1_g` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2013 | g    | B2    | 209942, 210514, 210603, 210639, 210675         | :numref:`source_counts_2013_B2_g` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2015 | g    | B2    | 427626                                         | :numref:`source_counts_2015_B2_g` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2013 | i    | B1    | 210631, 210667, 216988                         | :numref:`source_counts_2013_B1_i` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2015 | i    | B1    | 427616                                         | :numref:`source_counts_2015_B1_i` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2013 | i    | B2    | 210559, 210601, 210637, 210673, 216048         | :numref:`source_counts_2013_B2_i` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+   | 2015 | i    | B2    | 427624                                         | :numref:`source_counts_2015_B2_i` |
+   +------+------+-------+------------------------------------------------+-----------------------------------+
+
+Timing
+^^^^^^
+
+A final concern is the amount of time it will take to process each ccd in crowded fields.
+While a typical ccd took just under 4 minutes to process, there was a long tail of ccds that took far longer (:numref:`Timing_2013` and :numref:`Timing_2015`).
+The increased time was entirely spent in two steps: matching the detected objects to a reference catalog, and measuring the difference image sources.
+The time required for matching appeared to be non-linear, with the ccds with the largest number of sources and reference objects to match requiring up to four hours to complete.
+Our matching algorithm was not designed for these very large numbers of sources, so we are encouraged by the results even if the performance is slow. 
```

Comment:
The phrasing here was based on a conversation with Chris where he was surprised by the very high density of sources I was getting. I will change the wording to reflect that it was not optimized for these densities, instead of that it was not designed for them.